client/lib/sites-list/test/test-log-store.js

```diff
@@ -1,26 +1,35 @@
-var assert = require( 'assert' ),
-	actions = require( 'lib/mock-actions' );
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
 
-describe( 'Sites Log Store', function() {
-	var Dispatcher, LogStore;
+/**
+ * Internal dependencies
+ */
+import useMockery from 'test/helpers/use-mockery';
 
-	beforeEach( function() {
+describe( 'Sites Log Store', () => {
+	let Dispatcher, LogStore, actions;
+
+	useMockery();
```

Comment:
Yeah, I was following the instructions on the use mockery readme with `Dispatcher ` in mind. I'll run the tests without it and see if I get any errors